int16 id
geometry_msgs/Polygon poly
geometry_msgs/Pose2D pose
